### PR TITLE
Fix: Backup json file not created

### PR DESCRIPTION
**Pull request by sadanandl**
### Checklist:
- [ ] My code is tested
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: inc/lib/onboarding/assets/src/steps/import-site/index.js [hunk 1]
```diff
@@ -564,6 +573,66 @@ const ImportSite = () => {
...
+						__( 'Taking settings backup failed.', 'astra-sites' ),
+						'',
+						error?.message,
+						'',
+						'',
+						text
+					);
+
+					return false;
+				}
```
Comment (jaieds) [inline inc/lib/onboarding/assets/src/steps/import-site/index.js:620]: We should not mix up `async await` with `Promises (then and catch)`.

```suggestion
					const response = JSON.parse( text );
					if ( response.success ) {
						percentage += 2;
						dispatch( {
							type: 'set',
							importPercent: percentage,
						} );
						return true;
					}
					throw response.data;
```

Comment (sadanandl) [reply inc/lib/onboarding/assets/src/steps/import-site/index.js:620]: We followed the previous structure. I think we can keep as same.

Comment (jaieds) [reply inc/lib/onboarding/assets/src/steps/import-site/index.js:620]: @sadanandl - Yes. I understood. But If we use `.catch` then we don't need to use the `try...catch ` block.

**Review by jaieds — APPROVED** [review]

**Comment by jaieds** [comment]
@sadanandl - Removed the duplicate code block.

**Review by sagarb90 — APPROVED** [review]

File: inc/lib/onboarding/assets/src/steps/import-site/index.js [hunk 2]
```diff
@@ -433,16 +434,24 @@ const ImportSite = () => {
 			importPercent: percentage,
 		} );
 
+		let backFileStatus = false;
```
Comment (sagarb90) [inline inc/lib/onboarding/assets/src/steps/import-site/index.js:437]:
```suggestion
		let backUpFileStatus = false;
```

File: inc/lib/onboarding/assets/src/steps/import-site/index.js [hunk 3]
```diff
@@ -433,16 +434,24 @@ const ImportSite = () => {
...
 		let resetCustomizerStatus = false;
 		let resetWidgetStatus = false;
 		let resetOptionsStatus = false;
 		let reseteTermsStatus = false;
 		let resetPostsStatus = false;
 
+		/**
+		 * Settings backup file store.
+		 */
+		backFileStatus = await performSettingsBackup();
```
Comment (sagarb90) [inline inc/lib/onboarding/assets/src/steps/import-site/index.js:447]:
```suggestion
		backUpFileStatus = await performSettingsBackup();
```

**Pull request merged**